bokeh/resources.py

```diff
@@ -425,10 +425,18 @@ class CSSResources(BaseResources):
 
     '''
 
+    def __init__(self, mode='inline', version=None, root_dir=None,
+                 minified=True, log_level="info", root_url=None,
+                 path_versioner=None, custom_css_files=None):
+        super(CSSResources, self).__init__(mode='inline', version=version,
+            root_dir=root_dir, minified=minified, log_level=log_level,
+            root_url=root_url, path_versioner=path_versioner)
+        self.custom_css_files = [] if custom_css_files is None else custom_css_files
+
     @property
     def css_files(self):
         files, _ = self._resolve('css')
-        return files
+        return files + self.custom_css_files
```

Comment:
I had mixed feelings yesterday about this  so it's here for now.. If user wants `inline` and wants to inject it's own css that in being server from another (his own?) CDN or whatever, why we should not support it?